app/controllers/common_platform_data_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommonPlatformDataController < ApplicationController
+  include Response
+
+  def index
+    @common_platform_data = CommonPlatformData.all
+  end
+  
+  def show
+    @common_platform_datum = CommonPlatformData.find(params[:id])
+    json_response(@common_platform_datum)
```

Comment:
I think here we need to pass the output of the method `@common_platform_data.transformed_common_platform_data` as we don't want to send a representation of CommonPlatformData itself - but just the transformed data.